### ISSUE TITLE
[Dependency] Fix requirements.txt

### ISSUE THREAD

**Issue by FusRoman** [astrolabsoftware/fink-science#293]
change in the requirements.txt

* ```-e git+https://github.com/emilleishida/fink_sn_activelearning.git@fdd9eda71f56d5738c31947302f0bddf78e26780#egg=fink_sn_activelearning``` 
into 
```-e git+https://github.com/emilleishida/fink_sn_activelearning.git@fdd9eda71f56d5738c31947302f0bddf78e26780#egg=actsnfink```

* ```-e git+https://github.com/b-biswas/kndetect@kndetect#egg=kndetect``` into ``` -e git+https://github.com/b-biswas/kndetect#egg=kndetect```

**Comment by JulienPeloton** [comment]
Yes, the reference file is actually at https://github.com/astrolabsoftware/fink-broker/blob/master/deps/requirements-science.txt
